perun-core/src/test/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_member_attribute_def_def_o365EmailAddresses_muTest.java

```diff
@@ -152,6 +153,13 @@ public void testCorrect() throws Exception {
 		classInstance.checkAttributeSemantics(session, member, attributeToCheck);
 	}
 
+	@Test
+	public void testCorrectSyntax() throws Exception {
```

Comment:
Either change the previous test method name to checkCorrectSemantics or merge these two test methods to one with name testCorrect()

Reply:
The name of the tests were changed to testCorrectSemantics().